SuffolkLITLab/courtformsonline.org
# Github actions CI build workflow

This PR starts our github actions CI workflow.  Currently it will just verify that the app builds properly. This PR also includes an initial run of lint/format so that we can add a check for code formatting in the future.

Major Changes
- Add ci.yaml to workflow
- Initial yarn format and yarn lint ran
- Downgrade eslint to 8 for compatibility with current next version

QA Steps
- App should build and run with no changes 
- github action build check will need to be checked on a subsequent PR after this is merged

Note, LoC and file change numbers are inflated due to the initial run of prettier and eslint
